docs/guides/component-testing/angular/examples.mdx

```diff
@@ -193,6 +193,192 @@ in the future
+  standalone: true,
+})
+export class TestComponent {
+  title = input.required<string>()
+  count = model.required<number>()
+}
+```
+
+```html
+<!-- app/components/test-component.component.html -->
+<p data-cy="test-component-title-display">{{ title() }}</p>
+<p data-cy="test-component-count-display">{{ count() }}</p>
+<button data-cy="test-component-count-incr" (click)="count.set(count() + 1)">
+  Increase
+</button>
+<button data-cy="test-component-count-decr" (click)="count.set(count() - 1)">
+  Decrease
+</button>
+```
+
+#### Testing Input Signals
+
+Having `input()` signals work outside the Angular injection context is a bit tricky. Input signals cannot be created outside the Angular injection context, nor is there a way to provide an injection context. Because of this, there are two ways to test `input()` signals with Cypress Component Testing.
+
+##### Infered Generic Type
+
+In the example below, the `title` prop being passed into our `TestComponent` is a generic type of the `input()` signal. In this example, that generic type is a `string`. Cypress wraps the generic value in a writable `signal()` and merges it into the prop. In other words:
+
+```typescript
+let titleProp = 'Test Component'
+cy.mount(TestComponent, {
+  componentProperties: {
+    title: titleProp,
+  },
+})
+```
+
+Under the hood in the `@cypress/angular-signals` test harness is really:
+
+```typescript
+cy.mount(TestComponent, {
+  componentProperties: {
+    title: signal('Test Component'),
+  },
+})
+```
+
+##### Writable Signal
+
+Since the infered generic type in our example above lives outside the Angular context, there is no way for Cypress to enforce the one-way databinding on context updates. In other words, updates to the `titleProp` variable will not update the prop being passed into `TestComponent`.
+
+To solve this problem, Cypress allows users to pass a writable `signal()` wrapped in the generic type as the prop. This allows us to test our one-way databinding.
+
+```typescript
+const myTitlePropAsSignal = signal('Test Component')
+cy.mount(TestComponent, {
+  componentProperties: {
+    title: myTitlePropAsSignal,
+  },
+})
+
+cy.get('the-title-element').should('have.text', 'Test Component')
+cy.then(() => {
+  // now set the input() through a signal to update the one-way binding
+  myTitlePropAsSignal.set('FooBar')
+})
+
+cy.get('the-title-element').should('have.text', 'FooBar')
+```
+
+###### Change spy for signal `input()`
+
+Since the prop in question is an `input()`, Cypress does not propagate changes to the `titleChange` spy.
+
+```typescript
+cy.mount(TestComponent, {
+  componentProperties: {
+    title: 'Test Component',
+    // @ts-expect-error
+    titleChange: createOutputSpy('titleChange'),
+  },
+})
+
+// some action that changes the title prop...
+
+// this assertion will not be true. Cypress will never emit change events for props that are `input()` signals
+cy.get('@titleChange').should('have.been.called')
```

Comment:
I'd caution against providing examples that don't work. Users inevitably come in, scan, and copy paste then open issues about how the example doesn't work.